Justfile

```diff
@@ -8,6 +8,7 @@ default: format check test
 
 install:
     pip install poetry==1.7.1
+    poetry config virtualenvs.in-project true --local
```

Comment:
Do we need to make this change in this PR? I'd prefer to handle this separately if possible.

Reply:
Sure. This can be taken out.